Assignment 6

```diff
@@ -5,47 +5,53 @@ import collatz.CollatzCalculatorGrpc;
 import collatz.CollatzRequest;
 import collatz.CollatzResponse;
 
-public class CollatzCalculatorServer {
+public class CollatzCalServer
+{
     private final int port;
     private final Server server;
 
-    public CollatzCalculatorServer(int port) {
+    public CollatzCalServer(int port) 
+{
         this.port = port;
         this.server = ServerBuilder.forPort(port)
-                .addService(new CollatzCalculatorImpl())
+                .addService(new CollatzCalImpl())
                 .build();
     }
 
-    public void start() throws Exception {
+    public void start() throws Exception 
+{
         server.start();
         System.out.println("Server started, listening on " + port);
         Runtime.getRuntime().addShutdownHook(new Thread(() -> {
             System.err.println("*** shutting down gRPC server since JVM is shutting down");
-            CollatzCalculatorServer.this.stop();
```

Comment:
This is important, you definitely want to stop the server on jvm shutdown, or the port that the server is running on may not get released right away (and then you'll have to wait 15+ minutes depending on your operating system's timeout).